client/pages/OriginalDataPoint/components/NationalClasses/NationalClass/NationalClass.tsx

```diff
@@ -6,14 +6,18 @@ import VerticallyGrowingTextField from '@client/components/VerticallyGrowingText
 // import ReviewIndicator from '@webapp/app/assessment/components/review/reviewIndicator'
 import { useTranslation } from 'react-i18next'
 // import { useCountryIso } from '@client/hooks'
-// import { useAppDispatch } from '@client/store'
-import { useOriginalDataPoint } from '@client/store/pages/originalDataPoint'
+import { useAppDispatch } from '@client/store'
+import { useAssessment, useCycle } from '@client/store/assessment'
+import { OriginalDataPoint } from '@meta/assessment/originalDataPoint'
+import { OriginalDataPointActions, useOriginalDataPoint } from '@client/store/pages/originalDataPoint'
+import { deleteNationalClass } from '@meta/assessment/originalDataPoint/odps/deleteNationalClass'
+import { updateNationalClass } from '@meta/assessment/originalDataPoint/odps/updateNationalClass'
```

Comment:
use import ODPs 
and use ODPs.deleteNationalClass and ODPs.updateNationalClass
do not use individual methods